src/Compilers/CSharp/Test/Semantic/IOperation/IOperationTests_IUsingStatement.cs

```diff
@@ -0,0 +1,675 @@
+﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+
+using Microsoft.CodeAnalysis.CSharp.Syntax;
+using Microsoft.CodeAnalysis.Test.Utilities;
+using Xunit;
+
+namespace Microsoft.CodeAnalysis.CSharp.UnitTests
+{
+    public partial class IOperationTests : SemanticModelTestBase
```

Comment:
>IOperationTests [](start = 25, length = 15)

It would be good to have tests that directly request nodes for constituents of using statement: UsingStatementSyntax.Expression, UsingStatementSyntax.Statement, UsingStatementSyntax.Declaration, VariableDeclarationSyntax, VariableDeclaratorSyntax.

Reply:
Added.